bot/event-sub-handlers/channel-point-event.handler.ts

```diff
@@ -0,0 +1,27 @@
+import { EventSubChannelRedemptionAddEvent } from '@twurple/eventsub-base';
+import { inject, injectable } from 'inversify';
+import winston from 'winston';
+import InjectionTypes from '../../dependency-management/types';
+import ChannelPointRedeem from '../../database/models/channelPointRedeem.dbo';

```

Comment:
This should actually reference the db/index export instead of a direct reference